mne_features/tests/test_univariate.py

```diff
@@ -178,6 +179,40 @@ def test_pow_freq_bands():
                                                ratios='only'), expected_ratios)
 
 
+def test_feature_names_pow_freq_bands():
+    _data = data[:, :3, :]  # keep only 3 channels for the sake of simplicity
+    selected_funcs = ['pow_freq_bands']
+    fb = np.array([[4., 8.], [30., 70.]])
+    ratios_col_names = ['ch0_0_1', 'ch0_1_0', 'ch1_0_1', 'ch1_1_0',
+                        'ch2_0_1', 'ch2_1_0']
+    pow_col_names = ['ch0_0', 'ch0_1', 'ch1_0', 'ch1_1', 'ch2_0', 'ch2_1']
```

Comment:
no way to have more explicit names likes alpha, beta etc?

Reply:
We could improve that but, at some point, the user would need to name the frequency bands he wishes to use. What about allowing the `freq_bands` parameter in `compute_pow_freq_bands` to be a dict as the one below? 
```
freq_bands = {'delta': [0.5, 4], 
              'theta': [4, 8], 
              'alpha': [8, 13], 
              'beta': [13, 30], 
              'low-gamma': [30, 70], 
              'high-gamma': [70, 100]}
```